src/components/Header/styles.ts

```diff
@@ -13,7 +13,7 @@ export const Container = styled.View`
 `;
 
 export const IconContainer = styled.TouchableOpacity`
-  right: 50px;
+  right: 200%;
```

Comment:
Just to remind you that we should analyze the responsiveness of this icon later.

Reply:
It shouldn't even be there lol, I ended up commit accidentally sry